pythonforandroid/recipes/libffi/__init__.py

```diff
@@ -14,14 +14,11 @@ class LibffiRecipe(Recipe):
         - `libltdl-dev` which defines the `LT_SYS_SYMBOL_USCORE` macro
     """
     name = 'libffi'
-    version = '3.2.1'
-    url = 'https://github.com/libffi/libffi/archive/v{version}.tar.gz'
+    # Version pinned to post `v3.3RC0`
+    version = '8fa8837'
```

Comment:
minor: I'm surprised we need to be that cutting edge. What is this solving that the `3.2.1` or `3.3-rc0` isn't solving?

Reply:
**v3.2.1** it's from year 2014...it's a little outdated and has problems with `clang` (#1525)
**v3.3-rc0** it was released at april 2018 (it's a pre-release) and it lacks some commits that we are interested, if I remember well, we are specially interested in those ones that fixes specific issues for Arm64, you can see it in the page below searching for the commit I pinned.But honestly, I did this some months ago and I don't exactly remember what was the problem...

See also: https://github.com/libffi/libffi/commits/master

Reply:
Clear, thanks for clarifying @opacam

Reply:
@opacam Might be worth putting that info in a comment, can't hurt. But no worries either way.